runtime/runtime/src/lib.rs

```diff
@@ -52,12 +54,39 @@ mod system;
 pub const ETHASH_CACHE_PATH: &str = "ethash_cache";
 pub(crate) const POISONED_LOCK_ERR: &str = "The lock was poisoned.";
 
+/// Number of epochs it takes to unstake.
+const NUM_UNSTAKING_EPOCHS: BlockIndex = 3;
+
+/// Check that account still has enough amount to pay rent.
+fn check_rent(
+    account_id: &AccountId,
+    account: &mut Account,
+    runtime_config: &RuntimeConfig,
+    epoch_length: BlockIndex,
```

Comment:
Apparently, `epoch_length` is tricky, but why can't we put it into `RuntimeConfig`?

Reply:
Because it's not configuration of runtime but of a chain and we were planning to make it changeable via on chain voting over time